workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/movies.py

```diff
@@ -185,16 +234,21 @@ def main(args):
     prognostic = derived_variables.physics_variables(
         load_diags.SegmentedRun(args.url, catalog).data_2d
     )
+    verification = derived_variables.physics_variables(
+        get_verification(args, catalog, join_2d="inner").data_2d
+    )
     # crashed prognostic runs have bad grid vars, so use grid from catalog instead
     prognostic = (
         prognostic.drop_vars(GRID_VARS, errors="ignore")
         .drop_dims(INTERFACE_DIMS, errors="ignore")
         .merge(grid)
     )
+    merged = _merge_prognostic_verification(prognostic, verification).merge(grid)
 
     if args.n_timesteps:
-        max_time = min(args.n_timesteps, prognostic.sizes["time"])
-        prognostic = prognostic.isel(time=slice(None, max_time))
+        prognostic = _limit_time_length(prognostic, args.n_timesteps)
+        merged = _limit_time_length(merged, args.n_timesteps)
 
-    for name, movie_spec in _movie_specs().items():
-        _create_movie(name, movie_spec, prognostic, args.output, args.n_jobs)
+    for movie_spec in _MOVIE_SPECS:
+        dataset = merged if movie_spec.require_verification_data else prognostic
```

Comment:
I did it this way because they might have different time coordinates. E.g. for the 'standard' verification data, there is only 3-hourly water vapor path output, but I still want the movie of ML tendencies (which uses only prognostic data) to have 15-minute snapshots if available.